Makefile.PL

```diff
@@ -1,15 +1,15 @@
 use strict;
 use warnings;
 require 5.008 ;
+use utf8;
 
 use lib qw(inc);
 use ExtUtils::MakeMaker 5.16 ;
 use Config;
-use FindBin qw/$Bin/;
 use Devel::CheckLib;
 
 my $ofiles = '';
-foreach ( <$Bin/*.c> ){
+foreach ( <*.c> ){
```

Comment:
See the commit message that went with it - using FindBin adds no value here since the current directory must always be the root of the repository for the distribution to build. Therefore, `$Bin` is always `.` and can be safely inlined.